rosidl_generator_py/resource/_msg_support.entry_point.c.em

```diff
@@ -4,15 +4,33 @@
 #include <Python.h>
 #include <stdint.h>
 
-#include <rosidl_generator_c/message_type_support.h>
+@{
+static_includes = {}
+for spec, subfolder in message_specs:
+  if subfolder == 'msg':
+    static_includes[subfolder] = '#include <rosidl_generator_c/message_type_support.h>'
+  elif subfolder == 'srv':
+    static_includes[subfolder] = '#include <rosidl_generator_c/service_type_support.h>'
+for value in sorted(static_includes.values()):
+  print(value)
```

Comment:
nitpick: I prefer to see the code templated it out, i.e. `@[for value in sorted(static_includes.values())]@...`, rather than using `print` within the `@{}@`. But that's just my preference, as I tend to skip over `@{}@` sections when reading, assuming they are just setting up context. You don't have to change this unless you also think it's a good idea.